base/range.jl

```diff
@@ -1087,3 +1087,13 @@ julia> mod(3, 0:2)
 """
 mod(i::Integer, r::OneTo) = mod1(i, last(r))
 mod(i::Integer, r::AbstractUnitRange{<:Integer}) = mod(i-first(r), length(r)) + first(r)
+
+"""
+    clamp(x::Integer, r::AbstractUnitRange)
+
+Clamp `x` to lie within range `r`.
+
+!!! compat "Julia 1.6"
+     This method requires at least Julia 1.6.
+"""
+clamp(x::Integer, r::AbstractUnitRange{<:Integer}) = clamp(x, first(r), last(r))
```

Comment:
You need to add this inside the `Math` module in `base/math.jl`, since Base just imports `clamp` from `Math` and reexports it, so this just shadows `clamp` and doesn't overload it.

Reply:
Oh that may explain all the bizarre test failures, thanks.